### ISSUE TITLE
How to build outside of the build pipelines?

### ISSUE THREAD

**Issue by moonpiedumplings** [kasmtech/KasmVNC#153]
Hi, I would like to package kasmvnc for Nixos, but I am really struggling to understand how to package this. I've been going through your scripts for a few days now, and I can't figure anything out. 

The main reason why I want to sidestep your build pipelines is because nix packages many of the things that you statically link, or compile and install, like [libjpeg-turbo](https://search.nixos.org/packages?channel=22.11&show=libjpeg&from=0&size=50&sort=relevance&type=packages&query=libjpeg), which is actually the default version offered, or webp.

**Comment by mmcclaskey** [comment]
There are so many operating systems to support and many use outdated libjpeg-turbo and webp libs. We want the best possible performance and therefore we decided to statically link them. And while some operating systems might use the latest libjpeg-turbo today, that will not be true tomorrow. Webp actually has newer versions available, we upgraded as far as we could without breaking. When we fix what breaks us on the latest webp version, we will be running a version later than what is available on whatever nix flavor you happen to favor. The webp lib is actually fast evolving.

If you desire to tie yourself to whatever version of packages are available on your particular favored nix flavor, you may do so. Our entire build system is under the builder directory and the README there is self explanatory. You can run that directly, pipeline not required. You would have modify our build setup to dynamically link libraries, but if you go back a version we did not previously statically link libjpeg-turbo.

**Comment by moonpiedumplings** [comment]
No, your README is not self explanatory.  I've read it all, and I've been going through your bash scripts, and docker files, and I still can't figure out what your actual build command is. I'm guessing its cmake or make, but `make servertarball`, and `cmake .` , but both error when ran independently. 

NixOS is more than just another linux distro, it's also a build system, and OS agnostic package manager. But the disadvantage, is that packages must be built with Nix's tooling, which is incompatible with what you have.

By build pipeline, I meant those scripts and dockerfiles, and I would essentially like to port your build system from kasm to Nix, so I can use KasmVNC to manage my NixOS servers.

So I ask again, how do I build kasmvnc outside of that build pipeline? What is the actual build command used to get the binaries, from the source code?
